packages/lesmis/src/components/FavouritesMenu.js

```diff
@@ -5,9 +5,10 @@
  */
 
 import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
 import styled from 'styled-components';
-import { Typography, IconButton, Drawer, List, ListItem } from '@material-ui/core';
-import { Close, Favorite } from '@material-ui/icons';
+import { Typography, IconButton, Drawer } from '@material-ui/core';
+import { Close, StarBorder } from '@material-ui/icons';
 import { FlexStart } from './Layout';
 
 /** =================================
```

Comment:
This is just a placeholder component and doesn't need review at this time. There is another user story which covers it.

Reply:
Can we remove it for now? It's not part of MVP, so it's probably best for it not to be released with potentially confusing placeholder info